syntax = "proto3";

message UploadRequest {
  string apkfile_name = 1;
  int64 apkfile_size = 2;
  bytes apk_content = 3;
}

message UploadResponse {
  string message = 1;
}

service YourService {
  rpc Upload (UploadRequest) returns (UploadResponse);
}
